### PR TITLE
CIQUAL ericwang

### PR DESCRIPTION
**Pull request by gnaweric**
**Checklist:**

Make sure you've done all the following (You can delete the checklist before submitting)

* [x] Code is well documented
* [x] Include unit tests for new functionality
* [ ] Code passes Travis builds in your branch
* [ ] If you have multiple commits please combine them into one commit by squashing them.
* [ ] Read and understood the [contribution guidelines](https://github.com/openfoodfacts/openfoodfacts-server/blob/master/CONTRIBUTING.md)

**Description:**

Describe the changes made and why they were made instead of how they were made.

**Related issues and discussion:** #[ISSUE NUMBER]

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -34541,8 +34544,25 @@ zh:猪肉餐
 <en:Pork meals
 <en:skewers
 en:pork skewers
-fr:Brochettes de porc, brochette de porc
-nl:Varkensspiesjes
+fr:Brochettes de porc, brochette de porc, échine de porc rôtie, échine de porc au four
```
Comment (stephanegigandet) [inline taxonomies/categories.txt]: Here we want only synonyms, and "échine de porc au four" is not a synonym for "brochettes de porc".

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -34541,8 +34544,25 @@ zh:猪肉餐
...
+nl:Varkensspiesjes 
+ciqual_food_code:en:28451
+ciqual_food_name:en:Pork, loin, roasted/baked
+ciqual_food_name:fr:Porc, échine, rôtie/cuite au four
+
+<en: Pork meals
+en:Pork ham escalope cooked
+fr:porc cuit, escalope de jambon
```
Comment (stephanegigandet) [inline taxonomies/categories.txt]: entries like "fr: abc, xyz" will make "xyz" a synonym for "abc", which is not the case here.

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -35894,11 +35914,16 @@ nl:Opgemaakte salades
 
 <en:Prepared salads
 <en:Meals with fish
-en:Tuna salad
+en:Tuna salad, Prepared mixed meat/fish canned drained salad
 de:Thunfischsalat
 fi:tonnikalasalaatti
-fr:Salades au thon
 nl:Tonijnsalades
+fr:Salades au thon, salade composée avec viande apertisée égouttée, salade composée avec poisson appertisée égouttée
```
Comment (stephanegigandet) [inline taxonomies/categories.txt]: If a CIQUAL category matches 2 or more different categories, then we need to apply the CIQUAL category to all those different OFF categories, but we should not make it one category on OFF. So we do want to apply this ciqual category to "tuna salads", but we don't want to make those 2 new entries synonyms of it.

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -35894,11 +35914,16 @@ nl:Opgemaakte salades
 
 <en:Prepared salads
 <en:Meals with fish
-en:Tuna salad
+en:Tuna salad, Prepared mixed meat/fish canned drained salad
```
Comment (teolemon) [inline taxonomies/categories.txt]:
```suggestion
en:Tuna salad
```

File: taxonomies/categories.txt (hunk 1 above)
Comment (gnaweric) [reply taxonomies/categories.txt]: Yes, you right, thanks

Comment (teolemon) [inline taxonomies/categories.txt]:
```suggestion
fr:Brochettes de porc, brochette de porc
```

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -34541,8 +34544,25 @@ zh:猪肉餐
 <en:Pork meals
 <en:skewers
 en:pork skewers
-fr:Brochettes de porc, brochette de porc
-nl:Varkensspiesjes
+fr:Brochettes de porc, brochette de porc, échine de porc rôtie, échine de porc au four
+nl:Varkensspiesjes 
+ciqual_food_code:en:28451
```
Comment (teolemon) [inline taxonomies/categories.txt]:
```suggestion

fr:échine de porc rôtie, échine de porc au four
ciqual_food_code:en:28451
```

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -21989,7 +22131,11 @@ hu:Ementáli
 it:Emmental, emmenthal, emmentaler
 nl:Emmentalers
 wikidata:en:Q932214
-oqali_family:en: fr:Fromages - Emmental
+oqali_family:en: fr:Fromages - EmmentalΩ
```
Comment (teolemon) [inline taxonomies/categories.txt:22134]:
```suggestion
oqali_family:en: fr:Fromages - Emmental
```

File: taxonomies/categories.txt [hunk 7]
```diff
@@ -30275,14 +30576,18 @@ nl:Diepvriesmaaltijden
 pl:Mrożone dania gotowe
 
 <en:Frozen foods
-<en:Fishes
-en:Frozen fishes
+<en:es
+en:Frozen es
```
Comment (teolemon) [inline taxonomies/categories.txt:30580]:
```suggestion
<en:Fishes
en:Frozen fishes
```

File: taxonomies/categories.txt [hunk 8]
```diff
@@ -33241,7 +33546,7 @@ nl:Vanillesauzen
 wikidata:en:Q1342574
 
 <en:Sauces
-en:Sauces for fishes, sauces for fish
+en:Sauces for es, sauces for
```
Comment (teolemon) [inline taxonomies/categories.txt:33549]:
```suggestion
en:Sauces for fishes, sauces for fish
```

File: taxonomies/categories.txt [hunk 9]
```diff
@@ -33257,7 +33562,7 @@ fr:Sauces à l'oseille, Sauce à l'oseille
 <en:Sauces
 fr:Sauces américaines
 
-<en:Sauces for fishes
+<en:Sauces for es
```
Comment (teolemon) [inline taxonomies/categories.txt:33565]:
```suggestion
<en:Sauces for fishes
```

**Comment by teolemon** [comment]
I'm closing the PR since the copy was merged.

**Pull request closed without merging**